astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -5,14 +5,13 @@
 import os
 import re
 import sys
+import numpy as np
+import numpy.testing as npt
+import pytest
 from contextlib import nullcontext
 from io import BytesIO
 from textwrap import dedent
```

Comment:
```suggestion
from contextlib import nullcontext
from io import BytesIO
from textwrap import dedent

import numpy as np
import pytest
from numpy.testing import assert_allclose, assert_equal
```
Frankly I have no idea what import order Ruff wishes to enforce, but the previous form was accepted, so apparently alphabetical order _within_ a block is fine...
I personally prefer the option of using `assert_*` directly; since the old custom functions will be removed entirely, I don't think there is much risk of confusion. But that is up for discussion.

Reply:
ruff is configured to the standard `isort` settings: first sort by package origin (standard library, third party, first party, local) then alphabetically within those blocks with `import X` preceding `from X import`.